hydroflow_lang/src/graph/ops/_lattice_fold_batch.rs

```diff
@@ -44,7 +44,7 @@ pub const _LATTICE_FOLD_BATCH: OperatorConstraints = OperatorConstraints {
     is_external_input: false,
     ports_inn: Some(|| PortListSpec::Fixed(parse_quote! { input, signal })),
     ports_out: None,
-    input_delaytype_fn: |_| Some(DelayType::Stratum),
+    input_delaytype_fn: |_| Some(DelayType::MonotoneAccum),
     flow_prop_fn: None,
```

Comment:
Should be `flow_prop_fn: Some(LATTICE_FOLD_REDUCE_FLOW_PROP_FN),` (for Cumul?) - this may fix the warning messages we looked at

Reply:
actually wont fix the warnings since this is a different operator